src/GitVersionTask/GitVersionTask.csproj

```diff
@@ -11,10 +11,12 @@
         <DevelopmentDependency>true</DevelopmentDependency>
 
         <LangVersion>8.0</LangVersion>
-    </PropertyGroup>
+    </PropertyGroup>   
```

Comment:
I do not like these random whitespace inconsistency left around in the project. 😭

Reply:
I wonder if this is something dotnet-format and editorconfig rules could catch in future.

Reply:
editorconfig should already trim whitespace: https://github.com/GitTools/GitVersion/blob/master/.editorconfig#L9

dotnet format possible could.

Reply:
dotnet-format docs mentions trailing whitespace isnt supported but states the tool removes trailing whitespace by default: https://github.com/dotnet/format/blob/master/docs/Supported-.editorconfig-options.md

Reply:
I'm not sure how the whitespace came about but I do have a habbit of ctrl + k + d to auto format documents in VS, but I would hope that should also honour editorconfig rules, so perhaps this was just a flourish of the fingers.